src/parse.js

```diff
@@ -4,11 +4,19 @@ import vcfReserved from './vcfReserved'
  * Class representing a VCF parser, instantiated with the VCF header.
  * @param {object} args
  * @param {string} args.header - The VCF header. Supports both LF and CRLF
+ * @param {boolean} args.strict - Whether to parse in strict mode or not
```

Comment:
This needs to move down a line so "newlines" from the line below goes with the `header` option.